lcm/property.go

```diff
@@ -13,6 +13,28 @@ const (
 	ProperNodes = "AvailableNodes"
 )
 
+const (
+	RegisterSecretForCache    string = "registrySecret"
+	ClairSecretForCache       string = "clairSecret"
+	ChartMuseumSecretForCache string = "chartMuseumSecret"
+	JobServiceSecretForCache  string = "jobServiceSecret"
+)
+
+const (
+	CoreSecretForDatabase         string = "coreSecret"
+	ClairSecretForDatabase        string = "clairSecret"
+	NotaryServerSecretForDatabase string = "notaryServerSecret"
+	NotarySignerSecretForDatabase string = "notarySignerSecret"
+)
+
+const (
+	AzureSecretForStorage string = "azureSecret"
```

Comment:
the name of the storage secret is same with [here](https://github.com/goharbor/harbor-cluster-operator/blob/master/controllers/storage/provision.go#L21), we should move to common constants in the future.